code/game/objects/items/reagent_containers/food/cans.dm

```diff
@@ -92,50 +92,18 @@
 
 
 
-//DRINKS
+//SODA
 
-/obj/item/reagent_container/food/drinks/cans/cola
-	name = "\improper Fruit-Beer"
-	desc = "In theory, Mango flavored root beer sounds like a pretty good idea. Weyland-Yutani has disproved yet another theory with its latest line of cola. Canned by the Weyland-Yutani Corporation."
-	icon_state = "fruit_beer"
+/obj/item/reagent_container/food/drinks/cans/classcola
```

Comment:
intended to be unobtainable?

Reply:
This is specific for admin use and map makers. There is no cola vendor that sells generic sodas like Classic Cola, only Souto vendors. I didn't want to add a new cola vendor to this PR.